lnrpc/rpc.proto

```diff
@@ -212,70 +442,109 @@ message SendResponse {
 
 message ChannelPoint {
     // TODO(roasbeef): make str vs bytes into a oneof
+
+    /// Txid of the funding transaction
     bytes funding_txid = 1 [ json_name = "funding_txid" ];
+    /// Hex-encoded string representing the funding transaction
     string funding_txid_str = 2 [ json_name = "funding_txid_str" ];
+    /// The index of the output of the funding transaction
     uint32 output_index = 3 [ json_name = "output_index" ];
 }
 
 message LightningAddress {
+    /// The identity pubkey of the Lightning node
     string pubkey = 1 [json_name = "pubkey"];
+    /// The network location of the lightning node, e.g. `69.69.69.69:1337` or `localhost:10011`
     string host = 2 [json_name = "host"];
 }
 
+/** `send-json-string` decodes addresses and the amount to send
```

Comment:
Ahh, that was from lncli. I modified the script now to include lncli descriptions